alembic/versions/merge_heads_third.py

```
@@ -1,27 +1,151 @@
+            postgresql.JSON(astext_type=sa.Text()),
+            nullable=False,
+            server_default=sa.text("'[]'::json"),
+        ),
+        sa.Column("assignee_type", sa.String(length=50), nullable=False, server_default="round_robin"),
+        sa.Column("assignee_id", sa.Integer(), nullable=True),
+        sa.Column("priority", sa.Integer(), nullable=False, server_default="0"),
+        sa.Column("is_active", sa.Boolean(), nullable=False, server_default=sa.true()),
+        sa.Column("created_at", sa.DateTime(timezone=True), server_default=sa.text("now()"), nullable=False),
+        sa.Column("updated_at", sa.DateTime(timezone=True), server_default=sa.text("now()"), nullable=False),
+        sa.PrimaryKeyConstraint("id"),
+        sa.ForeignKeyConstraint(["tenant_id"], ["tenants.id"]),
```

Pylint (4.0.5)

[error] Undefined variable 'sa'

(E0602)